config/vufind/KohaRest.ini

```diff
@@ -184,3 +184,9 @@ extraFields = item-issue:acceptTerms:pickUpLocation
 [ItemTypeRenewalBlockMappings]
 ; ILL[too_many] = "ILSMessages::ill_item_renewal_limit"
 ; ILL[too_soon] = "ILSMessages::ill_renewal_too_soon"
+
+; This section controls how the location field in the holdings/status results is populated.
+; It can either be Branch (Koha library branch/physical location) or Shelving (Koha
+; permanent shelving location of an item)
+[LocationFieldForHoldingStatus]
```

Comment:
Looking at this with fresh eyes raises a question: Why isn't this just a single setting in the Holdings section?

Reply:
thanks @EreMaijala , I've added this in my latest follow-up